pkg/proxy/proxy.go

```diff
@@ -5,18 +5,20 @@ import (
 	"net"
 )
 
+// Proxy defines the behavior of a proxy.
```

Comment:
I've tried to complete it a bit based on how I saw it use for now : https://github.com/docker/libnetwork/blob/master/portmapper/proxy.go#L43. Not sure that's enough though :blush:.